web-console/knikubevirt/Create-vm/create-vm.md

```diff
@@ -0,0 +1,189 @@
+# Create A Virtual Machine
+
+## Launching the Wizard
+
+![Create VM](img/create-vm.png)
+
+When no virtual machines exist, the Virtual Machines section of the UI will include quick access to the Create Virtual Machine wizard in its empty state.
+
+
+## Step 1: Basic
+
+![Basic step](img/Step-1-basic-template.png)
```

Comment:
The only thing with the auto generated name is that it might not always be generated. There is a  possibility that the feature has been disabled OR they are creating a VM from scratch and haven't chosen any settings yet. This is the only one I left it empty though for that reason. There will be a follow up PR to focus on this feature specifically.